pkg/comment/comment_message_loader.go

```diff
@@ -0,0 +1,104 @@
+package comment
+
+import (
+	"bytes"
+	"fmt"
+	"net/url"
+	"regexp"
+	"text/template"
+
+	assets "github.com/arquillian/ike-prow-plugins/pkg/assets/generated"
+	"github.com/arquillian/ike-prow-plugins/pkg/config"
+	"github.com/arquillian/ike-prow-plugins/pkg/github/service"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/utils"
+)
+
+// Message struct for the plugin comment service
+type Message struct {
+	Thumbnail      string
+	Description    string
+	ConfigFile     string
+	Documentation  string
+	MessageFileURL string
+}
+
+var commentMessage Message
```

Comment:
I guess, that this should be used either as a function parameter or the functions as a methods with this receiver. In otrer words, don't use it as a global variable

Reply:
Looking at the line:
https://github.com/arquillian/ike-prow-plugins/pull/171/files#diff-1d8831f0937e79f278010cd60a0e253cR31
the method with a receiver would be probably better